handler.js

```diff
@@ -16,33 +17,39 @@ const probot = createProbot({
 })
 
 // Load Probot application
-probot.load(todo)
+probot.setup([todo])
```

Comment:
This is what I was missing and what we should probably document in the lambda handler. Is the difference that `probot.load` overrides the default plugins, while `probot.setup` allows you to add other plugins?

Reply:
`probot.setup()` calls `probot.load()` on an array of default plugins *and* whatever other plugins you pass it

Reply:
Gotcha. So calling `probot.load`, it never even looks for the default plugins.